core-services/src/main/java/org/zalando/nakadi/repository/kafka/KafkaRecordDeserializer.java

```diff
@@ -11,8 +12,9 @@ public class KafkaRecordDeserializer implements RecordDeserializer {
 
     private final AvroDeserializerWithSequenceDecoder decoder;
 
-    public KafkaRecordDeserializer(final SchemaProviderService schemaService) {
-        this.decoder = new AvroDeserializerWithSequenceDecoder(schemaService);
+    public KafkaRecordDeserializer(final SchemaProviderService schemaService,
```

Comment:
Maybe taking `AvroDeserializerWithSequenceDecoder` as constructor argument is better in this case as it reduces the dependencies on `SchemProviderService` and `LocalSchemRegistry`. 
But its okay to keep this as it is as well.